Octokit.Tests/Clients/SearchClientTests.cs

```diff
@@ -206,7 +206,7 @@ public void TestingTheLanguageQualifier()
                 client.SearchUsers(request);
                 connection.Received().Get<SearchUsersResult>(
                     Arg.Is<Uri>(u => u.ToString() == "search/users"),
-                    Arg.Is<Dictionary<string, string>>(d => d["q"] == "github+language:Ruby"));
+                    Arg.Is<Dictionary<string, string>>(d => d["q"] == "github+language:ruby"));
```

Comment:
I'm surprised I had to change that since the `MergedQualifiers` method called `Language.ToParameter()` and since `Language.Ruby` didn't have a `Parameter` attribute, it should have defaulted to the `ToLowerInvariant` version of `Ruby`. Am I missing something?